gnomad/utils/vcf.py

```diff
@@ -1482,29 +1482,29 @@ def make_hist_dict(
     return header_hist_dict
 
 
-def set_female_y_metrics_to_na(
+def set_xx_y_metrics_to_na(
     t: Union[hl.Table, hl.MatrixTable],
 ) -> Dict[str, hl.expr.Int32Expression]:
     """
-    Set AC, AN, and nhomalt chrY variant annotations for females to NA (instead of 0).
+    Set AC, AN, and nhomalt chrY variant annotations for XX samples to NA (instead of 0).
 
-    :param t: Table/MatrixTable containing female variant annotations.
+    :param t: Table/MatrixTable containing XX variant annotations.
     :return: Dictionary with reset annotations
     """
     metrics = list(t.row.info)
-    female_metrics = [x for x in metrics if "_female" in x or "_XX" in x]
```

Comment:
This one already handles both, but I'm fine if we want to only support XX now

Reply:
this might have been a tab completion that I missed. I'll add the old code back just in case